### PR TITLE
Done my adventures

### PR DESCRIPTION
**Pull request by swissbusybee**
Created the ability to mark adventures as done and move them to another section.  **We need to discuss the clickable cards as I could not put the button in the card as it would not accept two clicks.  I highly suggest that we only allow clicks on image and title in the cards.**

### CODE REVIEW

**Comment by joyceykao** [comment]
@swissbusybee I had similar issues with the delete and edit icons, but how I ended up getting them onto the card was using CSS position (absolute/relative) feature to pin the buttons on the card. We can talk about it on Monday and/or I can also pull this branch and take look this weekend.

**Comment by swissbusybee** [comment]
Thanks but go ahead and merge this branch as the functionality to move cards is working just fine.    The formatting of the button can be done in a different branch if needed.

File: app/views/my_adventures/index.html.erb [hunk 1]
```diff
@@ -60,6 +65,59 @@
...
       </div>
 
+      <div class="card mt-4 mb-4 p-4">
+        <h2>MY COMPLETED ADVENTURES (<%= @done_adventures.count %>)</h2>
+        <%= "Go on some adventures!" if @done_adventures.empty?  %>
+
+        <% @done_adventures.each_with_index do |a, index| %>
+          <div class="d-inline-flex align-items-center">
+            <%= link_to adventure_path(a.adventure), class: "card-product", id: "card-link-text" do %>
+              <% if a.adventure.photo.key.present? %>
```
Comment (joyceykao) [inline app/views/my_adventures/index.html.erb]: I get an error here on line 75 when I try to load the my_adventures page. 
`Module::DelegationError in MyAdventures#index`

Error message says this:
`key delegated to attachment, but attachment is nil`

**Review by joyceykao — CHANGES_REQUESTED** [review]
I got the my_adventures page to load once and then got an error in line 75 after I clicked the Done button. Now the my_adventures#index doesn't load at all. See comment in my_adventures/index.erb.html file.

**Comment by swissbusybee** [comment]
Strange as it was working for me but I think I had an extra key.  Try this new push.

File: app/views/my_adventures/index.html.erb (hunk 1 above)
Comment (joyceykao) [reply app/views/my_adventures/index.html.erb]: The page works now and the button works as expected. :) Nice!

**Review by joyceykao — COMMENTED** [review]
This fixes the issue for me. :)

**Comment by joyceykao** [comment]
@swissbusybee One small thing I noticed was that there are edit and delete icons in on the completed adventure cards. I think they should be removed?
Screen Shot 2020-05-29 at 11 05 49 AM

**Comment by joyceykao** [comment]
@swissbusybee I ended up doing something wonky to my local master and ended up having to build on this branch so I'll just go ahead and merge the changes here since everything here is functional.

**Pull request merged**